Cargo.lock

```diff
@@ -2138,26 +2160,26 @@ dependencies = [
 
 [[package]]
 name = "impl-codec"
-version = "0.4.1"
+version = "0.4.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "parity-scale-codec 1.0.5 (registry+https://github.com/rust-lang/crates.io-index)",
+ "parity-scale-codec 1.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
I've noticed we're using this even though we probably shouldn't (this comes as part of default features of `primitive-types`), but I guess it's not substantial.

Reply:
I've seen it too, it's been in there for a while now. :/

Reply:
`codec` has been a included in the default features for a while now,

https://github.com/paritytech/parity-common/commit/79898165c4ef3444a3b00087839fbb63b1c63f39#diff-7b2119e2b4c3e0074b53b88c21461b62